tests/unittests/test_handler/test_schema.py

```diff
@@ -447,4 +448,27 @@ def test_all_integration_test_cloud_config_schema(self):
         if errors:
             raise AssertionError(', '.join(errors))
 
+
+class TestSchemaDocExamples:
+    @skipUnlessJsonSchema()
+    def test_schema_doc_examples(self):
+        ignored_files = [
+            'cloud-config-archive-launch-index.txt',
+            'cloud-config-archive.txt',
+        ]
+
+        examples_dir = Path(
+            __file__).parent.parent.parent.parent / 'doc' / 'examples'
```

Comment:
This file (or test) could move around, which would break this determination. Adding an `import cloudinit` and using `cloudinit.__file__` (which is `cloudinit/__init__.py`, to save you looking it up too) would (a) be more robust, and (b) require less `.parent`ing.